### PR TITLE
chore: move to github actions

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      - master
+  pull_request:
+
+jobs:
+    test:
+        runs-on: ${{matrix.operating-system}}
+        strategy:
+            matrix:
+                operating-system: [ ubuntu-latest ]
+                php: [ "5.6", "7.0", "7.1", "7.2", "7.3", "7.4" ]
```
Comment (jdpedrie) [inline .github/workflows/tests.yml:14]: Should we add PHP 8.0?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            - name: Install Dependencies
+              uses: nick-invision/retry@v1
+              with:
+                timeout_minutes: 10
+                max_attempts: 3
+                command: composer install
+            - name: Run Script
+              run: vendor/bin/phpunit
+
+    test_php55:
```
Comment (jdpedrie) [inline .github/workflows/tests.yml:31]: Maybe I'm blind, but is there any difference between this and the 5.4 test and the regular one? I remember there used to be with the old php setup action, but I can't see any now.

Comment (bshaffer) [reply .github/workflows/tests.yml:31]: No there isn't. I assumed that the matrix above didn't support 5.5 so we had to create a special one.

Comment (bshaffer) [reply .github/workflows/tests.yml]: (updated)

File: .github/workflows/tests.yml (hunk 1 above)
Comment (bshaffer) [reply .github/workflows/tests.yml]: I'd be ok holding off until it's released

Comment (bshaffer) [reply .github/workflows/tests.yml]: (mostly because it's failing right now)

Comment (jdpedrie) [reply .github/workflows/tests.yml]: fine with me then.

**Review by jdpedrie — APPROVED** [review]

**Pull request merged**